R/scripts.R

```diff
@@ -99,12 +107,33 @@ run_civis <- function(expr, ...) {
 #'
 #' # Run the template
 #' run_template(id, arguments = list(arg1 = 1, arg2 = 2), ...)
+#'
+#' # Run the template and return JSON value outputs
+#' run_template(id, arguments = list(arg1 = 1, arg2 = 2), JSONValue=TRUE...)
```

Comment:
missed a comma here:

`JSONValue=TRUE, ...`

Reply:
Also, for consistency this should be `JSONValue = TRUE`

Reply:
fixed it.